Solutions/FlareSystemsFirework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Data Connector should be restored back to actual Data Connector content, instead of duplicating Workbook content.

Reply:
Workbook updates should also be added in the actual workbook file instead.

Reply:
New workbook section "Total Leaked Credentials received" contains error, pictured below:
![image](https://user-images.githubusercontent.com/55556791/120562088-940ac800-c3ba-11eb-9a2b-0862eddd1712.png)

Reply:
> Data Connector should be restored back to actual Data Connector content, instead of duplicating Workbook content.

Duh... Copy paste gone wrong. Sorry! Fixed!


>Total Leaked Credentials received" contains error from missing field,

Good catch! This column is not present in many use cases, so we might as well adapt the query to be resilient if its absent.

Done.